delfin/task_manager/scheduler/schedulers/telemetry/failed_telemetry_job.py

```diff
@@ -41,6 +41,22 @@ def __call__(self):
         :return:
         """
         try:
+            # Remove jobs from scheduler for deleted failed_tasks
+            # Remove jobs from scheduler if it is added
```

Comment:
Could not understand the comments

Reply:
updated comment . This block will check whether the task is marked for delete, if yes remove the associated job from scheduler and then remove the entry from DB.